tests/test_integration/conftest.py

```diff
@@ -342,7 +342,6 @@ def upsert_transformation_with_run(
     return created
 
 
-@pytest.mark.usefixtures("aggregator_two_datasets")
```

Comment:
[Does not have effect](https://docs.pytest.org/en/stable/deprecations.html#applying-a-mark-to-a-fixture-function)